client/src/components/page/Sidebar/index.tsx

```diff
@@ -63,6 +63,20 @@ const defaultSidebarProps: Partial<SidebarProps> = {
 export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) => {
   const { extraData } = props;
   const { roles } = extraData;
+  const [openKeys, setOpenKeys] = useState(['admin']);
+  let location = useLocation();
+
+  useEffect(() => {
+    setOpenKeys(['admin']);
```

Comment:
we need to find a better way to set open menus since we want to avoid having to update this block each time we have a new module